net/mini_snmpd/Makefile

```diff
@@ -8,46 +8,58 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mini_snmpd
-PKG_VERSION:=1.2b
-PKG_RELEASE:=8
+PKG_VERSION:=1.4-rc1
+PKG_RELEASE:=3
+PKG_MAINTAINER:=Luke McKee <hojuruku@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
 
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/troglobit/mini-snmpd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=203d92e60ed09466d6676c6ad20ad6cb2ce08a5d
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://members.aon.at/linuxfreak/linux/
-PKG_MD5SUM:=9e432c50ba8216d7fab0983b11b7112a
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
+# As warned by upstream maintainer and gcc
+PKG_SSP:=0
+PKG_RELRO:=0
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/mini-snmpd
   SECTION:=net
   CATEGORY:=Network
-  TITLE:=SNMP server for embedded systems
-  URL:=http://members.aon.at/linuxfreak/linux/mini_snmpd.html
+  TITLE:=A tiny SNMP server for embedded systems 
+  URL:=http://troglobit.github.io/mini-snmpd.html
+# uncomment if you just want the binary, not the init script
+# openwrt requires init script runtime dependencies be defined for make menuconfig 
+# (e.g busybox sysntpd)
+  DEPENDS:=+jsonfilter +ubus +procd +ubox
 endef
 
-ifneq ($(CONFIG_IPV6),)
-  TARGET_CFLAGS+= -D__IPV6__
-else
-  TARGET_CFLAGS+= -D__IPV4__
-endif
+CONFIGURE_ARGS+= \
+	$(if $(CONFIG_IPV6),,--disable-ipv6)
 
-TARGET_CFLAGS+= -DSYSLOG
+# Configure weirdness - Disabled by default, explicitately disabling turns feature on!
+#     --disable-debug \
+#	--disable-demo - Upstream Github Issue #4 Fixed 20160707
 
-MAKE_FLAGS+= \
-	OFLAGS="$(TARGET_CFLAGS)" \
+CONFIGURE_VARS+= LD="$(TARGET_CC)" \
 	STRIP="/bin/true" \
-	INSTALL_ROOT="$(PKG_INSTALL_DIR)"
+	CFLAGS="$(TARGET_CFLAGS)"
+
+# FYI upstream suggested stripping options
+#	STRIP="strip -S --strip-unneeded --remove-section=.note.gnu.gold-version --remove-section=.comment --remove-section=.note --remove-section=.note.gnu.build-id --remove-section=.note.ABI-tag mini_snmpd" 
+MAKE_FLAGS+= STRIP="/bin/true" 
```

Comment:
I've made a build removing CONFIGURE_VARS and MAKE_FLAGS and size stays the same, I think PKG_FIXUP:=autoreconf is taking care of everything